model/job_queue/v1/job_resource.model

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages status of jobs on a job queue.
+resource Job {
+    // Mark a job as Successful
+    method Success {
+        ReceiptId Integer
+    }
+
+    // Mark a job as Failed
+    method Failure {
+        ReceiptId Integer
+        FailureReason String
```

Comment:
Same, add `in`, `out` or `in out`, whatever is appropriate.

Reply:
Added `in` and comments